CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Master
 
+## 0.7.0
+* Updated Readme
+* Updated Dependencies 
+* Ran Formatter 
+* Fixed warnings up to 1.8
+* Add OTP 21
```

Comment:
I don't see OTP 21 in the travis matrix, can you add that?

Reply:
removing for now

Reply:
👍